### ISSUE TITLE
chapter 2，Q9

### ISSUE THREAD

**Issue by wanghsinche** [ilmoi/MMLBook#11]
>Q: A = {(λ, λ + µ3, λ − µ3) | λ, µ ∈ R}
>A: This has a basis of {(1,1,1)T,(0,1,−1)T}, so it is a subspace of R3.

Hi, could you explain this or give me some reference? Why we can say V has a basis of xxx, so it is a subspace of R3?

**Comment by BorScho** [comment]
@wanghsinche 
If you take lambda * (1,1,1) + nyu * (0,1,-1) you will get all vectors of the form (lambda, lambda + nyu, lambda - nyu), i.e. the entire set A. Thus A is the set of all linear combinations of (1,1,1) and (0,1,-1) , i.e. A is the subspace generated by these two vectors.

**Comment by wanghsinche** [comment]
> @wanghsinche
> If you take lambda * (1,1,1) + nyu * (0,1,-1) you will get all vectors of the form (lambda, lambda + nyu, lambda - nyu), i.e. the entire set A. Thus A is the set of all linear combinations of (1,1,1) and (0,1,-1) , i.e. A is the subspace generated by these two vectors.

Thanks @BorScho  But there's still a bit of confusion to me. 
Does it mean that any vector space generated by two basis in R3 is a subspace of R3 ?

**Comment by BorScho** [comment]
@wanghsinche
Yes. A subspace, say S, of a vector space, V, is at first a subset of V (i.e. all vectors of the subspace have to be elements of the vector space). And in addition the vector-space axioms (p.26, definition 2.9 ) hold for S. That this is the case for the set of all linear combinations of two vectors from R3 has to be proved of course. Just take two vectors (x1, x2, x3) and (y1, y2, y3) and try to prove all the vector-space axioms listed in the definition. Once You did this for your self, you will see it immediately ;o).
As an exercise: prove that any line in R3 going through the origin, is a subspace of R3, but a line missing the origin is not (why?).
prove: the circle with radius=1 around the origin is not a subspace of R3 (which of the axioms does not hold?). If You have problems with these things, it will probably be usefull, to consult a text on linear algebra (either one of the texts metioned in the book or something like "linear algebra for engineers" - these texts tend to be less heavy on the purely theoretical side). I find the MML book to be vastly non-exhaustive, i.e. You cannot learn all of the topics from just this book and some of the excercises are not really solvable with just the information from the book itself. I have come to view this book as sort of a guide-line, what to learn for machine-learning and I am using the tips for "further reading" a lot... to know, what to learn and having an idea from where You can learn these things, is worth a ton of time!
